src/main/frontend/handler/editor.cljs

```diff
@@ -2749,15 +2749,28 @@
   [_state input input-id search-timeout]
   (fn [e key-code]
     (when-not (util/event-is-composing? e)
-      (let [k (gobj/get e "key")
-            code (gobj/getValueByKeys e "event_" "code")
-            format (:format (get-state))
-            current-pos (cursor/pos input)
+      (let [current-pos (cursor/pos input)
             value (gobj/get input "value")
             c (util/nth-safe value (dec current-pos))
+            [key-code k code is-processed?]
+            (if (and (mobile-util/native-android?)
+                     (or (= key-code 229)
+                         (= key-code 0)))
+              [(.charCodeAt value (dec current-pos))
+               c
+               (if (= c " ")
+                 "Space"
+                 (str "Key" (string/upper-case c)))
```

Comment:
Better use `cond` and check against alphabets.

And a bad case, when inputting `h3`

Reply:
I found it was hard to cover all chars here, like alphabets, digits, and special chars, even modifiers, so I only handled Space and String here. 
The `code` variable is mostly useful for desktop app, so it does not have a big impact on input on Android. (Test passed). 
Will add a branch for digits.